blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/MultiBranchPipelineImpl.java

```diff
@@ -206,6 +211,16 @@ public BluePipelineContainer getBranches() {
     }
 
     @Override
+    @Navigable
+    public BluePipelineContainer getFindBranches(final String type) {
```

Comment:
@vivek would appreciate taking a glance at these server-side changes to validate they make sense. I need to paginate PRs separate from branches on the server side.